LEAD_MAINTAINER_PROTOCOL.md

```diff
@@ -1,51 +1,72 @@
+    email: <Repo Lead Maintainer Email Address>,
+    username: <Github Username>
+  }
+  workingGroup: {
+    name: <Name of the Working Group that owns this Repo>,
+    entryPoint: <Url to the coordination entry point for the Working Group>
+  }
+}
+```
 
-**Lead Maintainer responsibilities:**
+**Repo Lead Maintainer responsibilities:**
 
 - Respect and follow the [IPFS Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
 - Have a complete understanding of the module purpose, its specification (if any) and how the module is used by other parts of the project.
 - Review and merge PRs.
 - Respond in a timely manner to Github issues and curate them to ensure that duplicates are coalesced, issues are labeled with difficulty and priority, and also tag any issue that is open for new contributors with the `help wanted` label.
-- Publish new versions of the module to npm.
+- Release new versions of the module/package (if a code Repo).
 
-**Lead Maintainer expected attitude:**
+**Repo Lead Maintainer expected attitude:**
 
 - Be proactive in increasing the quality of the module. This goes from improving documentation, tests, codebase and more.
 - Show a great level of rigor and polish in the code that they ship.
-- Help others in understanding how the module works and why it exists.
+- Help others in understanding how the module works (or repo) and why it exists.
 - Apply the established Contributing Guidelines to the project and help others do too
 
-**How to become a Lead Maintainer:**
+**How to become a Repo Lead Maintainer:**
 
-Currently, there is no formal test or request form to do so. Lead Maintainers will be invited and nominated by the Tech Lead once the expected behaviour and rigor is observed. A Lead Maintainer can nominate the next Lead Maintainer to the Tech Lead 
+Currently, there is no formal test or request form to do so. Repo Lead Maintainers will be invited and nominated by one of the IPFS Tech Leads and/or Working Group Captains once the expected behaviour and rigor is observed. A Repo Lead Maintainer can nominate the next Repo Lead Maintainer to the IPFS Tech Lead and/or Working Group Captain.
 
 **In practice:**
 
-- Update each package.json and README.md to have a `leadMaintainer` field.
-- Update packages table on each entry module (e.g https://github.com/ipfs/js-ipfs#packages) to also list the maintainer for each.
-- Protect the master branch and only grant permissions for merge to the Maintainer and the Tech Lead, only.
-- Grant publish permission to the Maintainer and Tech Lead (or another Tech Lead if it’s the same person), only.
+- Create a lead-maintainer.json and update the README.md to have a `Repo Lead Maintainer` section.
+- Protect the master branch and only grant permissions for merge to the Lead Maintainer and respective Working Group members.
+- If a software project, grant publish permission to the Lead Maintainer and Tech Lead (or another Tech Lead if it’s the same person), only.
```

Comment:
If in a position of great responsibility, 2FA should be enabled where applicable?

Reply:
We currently demand 2FA for everyone that has write access. It was a big thing, see https://github.com/ipfs/community/issues/263

Reply:
👍 I was thinking for npm (etc.) also

Reply:
Ah, good point!